src/pages/index.js

```diff
@@ -19,9 +19,9 @@ function HomepageHeader() {
         <Banner
             content={
               <div>
-                <Typography scale="body1">A scalable, on-demand compute engine in the cloud.</Typography> 
+                <Typography scale="body1">Your go-to hub for quickstarts, guides, tutorials, code samples, technical documentation, and downloads for all Teradata products.</Typography> 
```

Comment:
Can you make sure this is pulling from translations

Reply:
Done
<Banner
            content={
              <div>
                <Typography scale="body1">{translate({ message: 'developers.title_description' })}</Typography> 
                <br/>
                <Link to="https://quickstarts.teradata.com/">
                  <Button
                    label="Learn more"
                    primary={true}
                    icon="fa fa-arrow-right"
                    trailingIcon={true}
                  />
                </Link>
              </div>}
            imageSrc={HeroImageUrl}
            isHero={true}
            title={translate({ message: 'developers.welcome_message' })}
          />